syntax = "proto3";

package azureConnectorGrpcServer;
option go_package = "./";

service GrpcServer {
  rpc TransformData (stream Data) returns (stream Response) {};
}

message Data {
  bytes Data = 1;
}

message Response {
  string Response = 1;
  string ErrorMessage = 2;
  int32 StatusCode = 3;
  string RequestID = 4;
}

// protoc --go_out=. --go_opt=paths=source_relative --go-grpc_out=. --go-grpc_opt=paths=source_relative grpc_server.proto
